### PR TITLE
Issue #246 [Joomla! CMS Issue Tracker Item #25835] JTable:getInstance doesn't manage correctly multiple includePaths (#268)

### PR DESCRIPTION
**Pull request by tompap**
Issue #246 [Joomla! CMS Issue Tracker Item #25835] JTable:getInstance doesn't manage correctly multiple includePaths (#268)

On Staging

### CODE REVIEW

**Comment by ianmacl** [comment]
**WARNING! Pull request is not against staging!**

# Test Results

Total Tests: 1389
Assertions: 9867
Failures: 0
Errors: 0
Test Time: 185.481618

File: libraries/joomla/filesystem/path.php [hunk 1]
```diff
@@ -300,4 +300,50 @@ class JPath
...
+	
+	/**
+	 * Searches the directory paths for a given file.
+	 *
+	 * @param   mixed   $paths  An path string or array of path strings to search in
+	 * @param   string  $file   The file name to look for.
+	 *
+	 * @return  mixed   The full paths and files names for the target file, or boolean false if the file is not found in any of the paths.
+	 *
+	 * @since   11.2
```
Comment (elinw) [inline libraries/joomla/filesystem/path.php:312]: This should be 11.3.

**Comment by eddieajau** [comment]
Closing issue.  Please re-pull against the Joomla Platform's "staging" branch.  Thanks in advance.

**Comment by tompap** [comment]
I am not used with github :) I just tried to do that, I hope it's correct now. 
I need to learn and try to create a project on github to understand all stuff

**Pull request closed without merging**